### PR TITLE
fe: profile card

### PR DESCRIPTION
**Pull request by asseemm**
image

### CODE REVIEW

File: client/src/atoms/ProfiAva.jsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { colors } from '../base';
+import { Flexbox } from '@atoms';
+import { Text } from '@atoms';
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: кривые импорты

File: client/src/atoms/ProfiAva.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import { sizes } from '../base';
+
+
+const ProfiMainInfo = styled.div`
+  display: flex;
+//   flex-wrap: wrap;
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: так не работает, надо убрать

File: client/src/atoms/ProfiAva.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { colors } from '../base';
+import { Flexbox } from '@atoms';
+import { Text } from '@atoms';
+import { sizes } from '../base';
+
+
+const ProfiMainInfo = styled.div`
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: ProfileWrapper or InfoWrapper

File: client/src/atoms/ProfiAva.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  width: 100%;
+`;
+
+const ProfiColumn = styled.div`
+    display: ${({ display }) => display};
+    justify-content: center;
+    text-align: center;
+`;
+
+const CircleAva = styled.div`
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: ProfileImageBox

File: client/src/atoms/ProfiAva.jsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    border-radius: 150px;
+    border: 6px solid #C0FDB6;
+    box-shadow: 0 0 10px #C0FDB6;
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 5%;
+`;
+
+const ProfiAvaImg = styled.img`
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: ProfileImage

File: client/src/atoms/ProfiAva.jsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+const CircleAva = styled.div`
+    width: 130px;
+    height: 130px;
+    border-radius: 150px;
+    border: 6px solid #C0FDB6;
+    box-shadow: 0 0 10px #C0FDB6;
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    margin-bottom: 5%;
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: Try avoid using percentages, since they rely on their parent element

File: client/src/atoms/ProfiAva.jsx [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+                        <defs>
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: svgs should be in asset/icons folder. And there should be `index.js` file exporting those icons.

File: client/src/atoms/ProfiAva.jsx [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+        <ProfiMainInfo>
+            <ProfiColumn>
+                <CircleAva>
+                    <ProfiAvaImg src="https://www.famousbirthdays.com/faces/heeseung-image.jpg" alt="Описание изображения" />
+                </CircleAva>
+                <Text fontSize={sizes.large}>Name Surname</Text>
+                <Text fontSize={sizes.medium}>id: 24743</Text>
+            </ProfiColumn>
+            <ProfiColumn display='grid'>
+                <Flexbox direction="column" gap="3%">
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: remove relative units

File: client/src/atoms/ProfiAva.jsx [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+                            <filter id="filter0_f_11_81" x="0" y="-4" width="67" height="67" filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+                                <feFlood flood-opacity="0" result="BackgroundImageFix" />
+                                <feBlend mode="normal" in="SourceGraphic" in2="BackgroundImageFix" result="shape" />
+                                <feGaussianBlur stdDeviation="2" result="effect1_foregroundBlur_11_81" />
+                            </filter>
+                        </defs>
+                    </svg>
+                    <Text fontSize={sizes.Large}>14 одногруппников</Text>
+                </Flexbox>
+
```
Comment (eyesofdaveed) [inline client/src/atoms/ProfiAva.jsx]: whitespaces

File: client/src/atoms/Progress.jsx [hunk 10]
```diff
@@ -0,0 +1,68 @@
+
+import React from 'react';
+import styled from 'styled-components';
+import { Progress, Space } from 'antd';
+
+
+import { Flexbox, Text } from '@atoms';
```
Comment (eyesofdaveed) [inline client/src/atoms/Progress.jsx]: fix imports

File: client/src/atoms/Progress.jsx [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+import { colors, sizes } from '../base';
+
+
+export const ProfiMainInfo = styled.div`
```
Comment (eyesofdaveed) [inline client/src/atoms/Progress.jsx]: InfoWrapper

File: client/src/base/colors.js [hunk 12]
```diff
@@ -9,5 +9,6 @@ export const colors = {
   eventBlue: "#BEBDF9",
   notificationColor: "#E4F3FB",
   profileHeaderBG: "#3B4051",
-  test_taskCardBG: "#E5F5FC"
+  test_taskCardBG: "#E5F5FC",
```
Comment (eyesofdaveed) [inline client/src/base/colors.js:12]: camelCase

File: client/src/organism/ProfiCard.jsx [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+
+import { ProfiAva } from '@atoms';
+import { ProfiProgress } from '@atoms';
+import { Flexbox } from '@atoms';
+import { Text } from '@atoms';
+import { sizes } from '../base/index';
+import { colors } from '../base';
+import { baseApi } from '@api';
+import { API_METHODS } from '@api/enums';
+
```
Comment (eyesofdaveed) [inline client/src/organism/ProfiCard.jsx:10]: fix imports

File: client/src/organism/ProfiCard.jsx [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+export const ProfiCard = () => {
+    return (
+        <ProfiCardWrapper>
+            <Flexbox width='100%' justify='space-between'>
+                <Flexbox width='50%' justify='left'>
+                    <Flexbox direction="column" gap="3%" align="flex-start">
+                        <ProfiAva></ProfiAva>
+                    </Flexbox>
+                </Flexbox>
+
```
Comment (eyesofdaveed) [inline client/src/organism/ProfiCard.jsx]: whitespace

File: client/src/organism/ProfiCard.jsx [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+  padding: 2% 5%;
+  box-shadow: 1px 2px 8px 0px #00000026;
+  border-radius: 28px;
+  max-width: 100%;
+`;
+
+export const ProfiCard = () => {
+    return (
+        <ProfiCardWrapper>
+            <Flexbox width='100%' justify='space-between'>
```
Comment (eyesofdaveed) [inline client/src/organism/ProfiCard.jsx]: Use Grid here

Comment (eyesofdaveed) [reply client/src/organism/ProfiCard.jsx]: Create an atom named `GridBox`

File: client/src/organism/ProfiCard.jsx [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+                <Flexbox width='50%' justify='left'>
+                    <Flexbox direction="column" gap="3%" align="flex-start">
+                        <ProfiAva></ProfiAva>
+                    </Flexbox>
+                </Flexbox>
+
+                <Flexbox width='45%' justify='space-between' align="end">
+                    <ProfiProgress></ProfiProgress>
```
Comment (eyesofdaveed) [inline client/src/organism/ProfiCard.jsx]: <ProfiileProgess />

Comment (eyesofdaveed) [reply client/src/organism/ProfiCard.jsx]: <ProfileProgress />

File: client/src/organism/ProfiCard.jsx [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+
+
+export const ProfiCardWrapper = styled.div`
+  background-color: ${colors.profileHeaderBG};
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 5%;
+  padding: 2% 5%;
```
Comment (eyesofdaveed) [inline client/src/organism/ProfiCard.jsx]: Use fixed values for now

**Review by Bashkanitto — APPROVED** [review]

**Pull request merged**